Server/src/diagramcoordinator.erl

```diff
@@ -12,55 +12,67 @@ init(_Coordinator, _Did, {_, []}, _) -> no_messages;
 %Spawns and Initializes the diagram coordinator
 init(Coordinator, Did, {Classes, Messages}, Class_names) -> 
 	%Sending information that the Coordinator has been spawned. To be printed client-side.
+	io:format("no1"),
```

Comment:
There are some code here that are in Sebastians pull request: review twice?

Reply:
Had to pull his code to create this feature

Reply:
So his feature should probably be integrated first